redis-http-connector/main.go

```diff
@@ -29,34 +29,34 @@ func (conn redisConnector) consumeMessage(ctx context.Context) {
 		"KEDA-Source-Name":    {conn.connectordata.SourceName},
 	}
 
-	var messages []string
-	var listItr int64
 	forever := make(chan bool)
 
 	go func() {
-		listLength, err := conn.rdbConnection.LLen(ctx, conn.connectordata.Topic).Result()
-		if err != nil {
-			conn.logger.Fatal("Error in consuming queue: ", zap.Error(err))
-		}
-		for listItr = 0; listItr < listLength; listItr++ {
-			msg, err := conn.rdbConnection.LPop(ctx, conn.connectordata.Topic).Result()
+		for {
```

Comment:
We can close forever after for loop. 
So that if a for loop exists by any chance pod would stop working and get restarted.

Also, we are passing context. Maybe we can use an exit signal handler so that the process exits cleanly.